frontend/model/contracts/group.js

```diff
@@ -345,8 +345,8 @@ const leaveChatRoomAction = (state, chatRoomID, memberID, actorID, leavingGroup)
     ...extraParams,
     hooks: {
       onprocessed: () => {
-        const rootState = sbp('state/vuex/state')
-        if (memberID === rootState.loggedIn.identityContractID) {
+        const { ourIdentityContractId } = sbp('state/vuex/getters')
+        if (memberID === ourIdentityContractId) {
```

Comment:
If you want, I will revert all my changes regarding `ourIdentityContractID` getter. I think as long as we have `getter` that returns the identity contract ID of the current user, we should not try to get it from the `state`. That's because once we need to change something related to this, it would be fine to update only the `getter`, not all the codes using `state`.

Reply:
@Silver-IT I agree with you that it is an improvement, and normally I would say it's OK, but for @corrideat's sake, let's please revert it. He has made changes to how this is done in this PR and the changes here will cause a lot of problems for him.

Reply:
Okay. I will revert it.